helm-charts/mend-renovate-ce/templates/deployment.yaml

```diff
@@ -231,6 +247,14 @@ spec:
             - name: MEND_RNV_WORKER_EXECUTION_TIMEOUT
               value: {{ .Values.renovate.mendRnvWorkerExecutionTimeout | quote }}
             {{- end }}
+            {{- if .Values.renovate.mendRnvWorkerCleanup }}
+            - name: MEND_RNV_WORKER_CLEANUP
```

Comment:
it seems `MEND_RNV_WORKER_CLEANUP_DIRS` is missing in both CE and EE

Reply:
added  :+1: